src/components/explorer/IndividualInterpretations.js

```diff
@@ -0,0 +1,285 @@
+import React, { useCallback, useMemo, useState } from "react";
+import PropTypes from "prop-types";
+
+import { Button, Descriptions, Empty, Icon, Modal, Table, Typography } from "antd";
+
+import { useIndividualInterpretations } from "./utils";
+import "./explorer.css";
+import { individualPropTypesShape } from "../../propTypes";
+import OntologyTerm from "./OntologyTerm";
+import { Route, Switch, useHistory, useParams, useRouteMatch } from "react-router-dom/cjs/react-router-dom.min";
+import { GeneDescriptor } from "./IndividualGenes";
+import VariantDescriptor from "./IndividualVariants";
+
+
+export const VariantInterpretation = ({ variationInterpretation }) => {
+    const [modalVisible, setModalVisible] = useState(false);
+
+    const closeModal = () => setModalVisible(false);
+    return (
+        <Descriptions layout="horizontal" bordered={true} column={1} size="small">
+            <Descriptions.Item label={"ACMG Pathogenicity classification"}>
+                {variationInterpretation.acmg_pathogenicity_classification}
+            </Descriptions.Item>
+            <Descriptions.Item label={"Therapeutic Actionability"}>
+                {variationInterpretation.therapeutic_actionability}
+            </Descriptions.Item>
+            <Descriptions.Item label={"Variant Descriptor"}>
+                <Button onClick={() => setModalVisible(!modalVisible)}>
+                    {variationInterpretation.variation_descriptor.id}
+                </Button>
+                <Modal
```

Comment:
set `footer={null}` here to remove ok/cancel buttons, since just having the 'close' button in the top right / being able to click outside the modal is good enough here